src/landing/model_list.rs

```diff
@@ -166,12 +176,20 @@ impl Widget for ModelList {
         if store.search.keyword.is_none() {
             if moly_mofa::should_be_visible() {
                 items.push(Item::Header("Featured Agents"));
-                items.extend(agents.chunks(3).map(|chunk| Item::AgentRow {
-                    agents: chunk,
-                    margin_bottom: 15.0,
-                }));
-                if let Some(Item::AgentRow { margin_bottom, .. }) = items.last_mut() {
-                    *margin_bottom = 0.0;
+                let agents_availability = store.chats.are_agents_available();
+                match agents_availability {
+                    AgentsAvailability::NoServers => items.push(Item::NoAgentsWarning("Not connected to any MoFa servers.")),
```

Comment:
hmmm... i suppose these strings are the same as in `chat_history`, maybe you could add a `to_human_message()`-like function to `AgentsAvailability` if you want to dedup, but I also like messages defined in the UI itself so, don't mind, just thinking aloud